afterpay/src/main/kotlin/com/afterpay/android/model/CheckoutV3CashAppPay.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright Block, Inc.
+package com.afterpay.android.model
+
+import kotlinx.serialization.Serializable
+
+
+@Serializable
+data class CheckoutV3CashAppPay(
```

Comment:
this is the  equivalent of iOS's `CheckoutV3CashAppPayPayload` 

I chose to *not* reuse `AfterpayCashApp` inside here (I'd have to make it @Serializable) so I just use those fields individually

Reply:
I'm in favor of deprecating `AfterpayCashApp` then, to make it easier for merchant integration. Reduce the surface of what they can use to what they actually _should_ be using

Reply:
Not sure we want to do that. `AfterpayCashApp` is used for the "V2 Checkout" version of CashAppPay. Could still duplicate the `AfterpayCashApp` class and deprecate in favour of more consistent naming?

ie
```
@Deprecated(
    "Use the CheckoutV2CashAppPay class",
    ReplaceWith("CheckoutV2CashAppPay")
)
```

Reply:
yeah @ScottAntonacAP 's right it's still needed for V2

@ScottAntonacAP I have a big list of things to alias-rename and deprecate, this will be one of them =)